storefront/app/helpers/spree/filters_helper.rb

```
@@ -48,23 +52,27 @@ def products_count_for_filter(name, value_id)
 
     def filter_price_range
       @filter_price_range ||= begin
-        product_ids = products_for_filters_scope(except_filters: [:price]).ids
+        products = products_for_filters_scope(except_filters: [:price])
 
-        {
-          min: Spree::Price.for_products(product_ids).minimum(:amount) || 0.to_d,
-          max: Spree::Price.for_products(product_ids).maximum(:amount) || 0.to_d
-        }
+        Rails.cache.fetch(["price-range", products, current_currency], expires_in: 1.day) do
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)